feat: [CHAOS-2619]: Adds cf-app-stop fault docs (#3742)
harness/developer-hub: docs/chaos-engineering/technical-reference/chaos-faults/cloud-foundry/cloud-foundry.md

---
id: cloud-foundry
title: Chaos Faults for Cloud Foundry
---

<!-- Import statement for Custom Components -->

import FaultDetailsCard from "@site/src/components/ChaosEngineering/FaultDetailsCard";
import ExperimentListSection from "@site/src/components/ChaosEngineering/ExperimentListSection"
import { experiments } from "./experiments"

<!-- Heading Description -->
## Introduction

> Cloud Foundry faults disrupt the Cloud Foundry resources. This deteriorates the performance of the app for the duration of the chaos experiment.

Reviewer: ```suggestion
Cloud Foundry faults disrupt the functioning of Cloud Foundry resources. This deteriorates the performance of the app for the duration of the chaos experiment.
```

<!-- Experiment List and Search Bar (every experiment added below, need to be added in this file also) -->

<ExperimentListSection experiments={experiments} />

<FaultDetailsCard category="cloud-foundry">

### CF app stop

> CF app stop injects app stop chaos for a Cloud Foundry app.

Reviewer: ```suggestion
CF app stop stops a Cloud Foundry application for a specified duration.
```

> - Checks app resilience against abrupt stopping.

Reviewer: ```suggestion
- Checks resilience against abrupt stop of application components/microservices.
```

- Validates the effectiveness of disaster recovery and high availability of the app.

<accordion color="green">
<summary>Use cases</summary>

> - Checks app resilience against abrupt stopping.

Reviewer: ```suggestion
- Checks resilience against abrupt stop of application components/microservices..
```
